experimental/segmenter/build.rs

```diff
@@ -393,6 +395,26 @@ fn generate_rule_segmenter_table(file_name: &str, toml_data: &[u8], provider: &F
                                     }
                                 }
 
+                                LineBreak::PostfixNumeric => {
+                                    if p.name == "PO_EAW"
+                                        && (eaw.get(i) == EastAsianWidth::Fullwidth
+                                            || eaw.get(i) == EastAsianWidth::Ambiguous
+                                            || eaw.get(i) == EastAsianWidth::Wide)
+                                    {
```

Comment:
`nit`: you can factor out the getter to avoid doing it three times in a pessimistic case.